app/app.ts

```diff
@@ -102,14 +91,49 @@ export class ChemiscopeApp {
                     $.ajax({
                         type: 'GET',
                         url: `examples/${structure.data}`,
-                        // this is getting deprecated, but the best option for now
+                        // this is getting deprecated, but the best option until
+                        // we change loadStructure to be an async callback
```

Comment:
Deferred loading of the structures will not work with external URL, since the developer also needs to provide a callback on how to load the structures. 

Since deferred structures loading requires you to setup your own chemiscope instance (to provide said callback) anyway, I think that's fine. We can find a way to make them work if there is demand for it later.